File: SayiOkuma.swift
//  Created by Kağan on 15.01.2017.
//  Copyright © 2017 Kağan Utku Kılıçlı. All rights reserved.

import Foundation

print("bir sayi giriniz:")
//girilen değer ile integer türünden bir değişken oluşturulabilmiş ise bunu ekrana yazdırır.
if let sayi = Int(readLine(strippingNewline: true)!) {
    print("girdiginiz sayi: \(sayi)")
}

File: SayiOkuma.c
#include <stdio.h>
#include <conio.h>//windows kullananlar icin programi calistirdiginizda kapanmadan once bir tusa basilmasini beklemeyi sagliyor

int main(){
  printf("bir sayi giriniz: ");
  int girdi; //bir integer belirliyorum
  scanf("%d",&girdi); //scanf ile bir deger alip az once belirledigim integera atiyorum
  printf("girdiginiz sayi: %d",girdi); //degeri ekrana yazdiriyorum
  getch();//windows kullananlar icin
}
